docs/Testing-your-templates.md

```diff
@@ -1,222 +0,0 @@
-dotnet/templating repo includes the test framework (ProjectTestRunner) which allows automated testing for the custom templates. The source code is available in [tools subfolder](https://github.com/dotnet/templating/tree/main/tools/ProjectTestRunner).
```

Comment:
it would be good to move this doc to `ProjectTestRunner` so we have at least a bit of info what it is about.